app/(gcforms)/[locale]/(form administration)/form-builder/[id]/components/dialogs/MoreDialog/StrictValue.tsx

```diff
@@ -0,0 +1,41 @@
+import { useTranslation } from "@i18n/client";
+import { Checkbox } from "@formBuilder/components/shared/MultipleChoice";
+import { FormElement } from "@lib/types";
+
+export const StrictValue = ({
+  item,
+  setItem,
+}: {
+  item: FormElement;
+  setItem: (item: FormElement) => void;
+}) => {
+  const { t } = useTranslation("form-builder");
+  const checked = item.properties.strictValue;
+
```

Comment:
Probably need a type check here to return null if it's not a Combobox component (see FormattedDateOptions or AddressCompleteOptions)

Reply:
added the check